src/cli/ops/console_mode.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * This file is part of clr-boot-manager.
+ *
+ * Copyright © 2016-2018 Intel Corporation
+ *
+ * clr-boot-manager is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1
+ * of the License, or (at your option) any later version.
+ */
+
+#include <ctype.h>
+#include <getopt.h>
+#include <string.h>
+
+#include "bootman.h"
+#include "cli.h"
+#include "update.h"
+
+// See `man loader.conf`
+static inline bool is_console_mode(const char *str)
+{
+        char list[][5] = {"", "0", "1", "2", "auto", "max", "keep"};
+
+        for(size_t i = 0; i < sizeof list; i++) {
+                if (strcmp(str, list[(int)i]) == 0) {
+                        return true;
+                }
+        }
+
+        return false;
+}
+
+bool cbm_command_set_console_mode(int argc, char **argv)
```

Comment:
For the benefit of looking back at this project for blsforme work could we return to using doxygen style comments for all non static APIs?